docs/DeveloperGuide.md

```diff
@@ -965,7 +946,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Appendix: Instructions for manual testing**
+## **Appendix B: Planned Enhancements**
+
+As we continue to develop PROperty, we are focused on refining and expanding its capabilities to better meet user needs. 
+Through identifying current limitations, we’re able to target areas that can benefit most from enhancement. 
+This section highlights our planned improvements, showcasing the steps we're taking to deliver a more comprehensive and effective solution for property agents.
+
+
+### Enhance Phone Numbers to Accept Special Characters
```

Comment:
Maybe we could enumerate the title like the example they gave us (see screenshot above).

Reply:
okies will do